app/javascript/packages/document-capture/components/selfie-capture.scss

```diff
@@ -63,6 +63,7 @@ div.selfie-capture__label[tabindex="-1"]:focus { // scss-lint:disable Qualifying
 .selfie-capture__consent-overlay-button {
   appearance: none;
   background: none;
+  cursor: pointer;
```

Comment:
looks like scss-lint doesn't like something about the order. Maybe it wants alphabetized?

> Properties should be ordered appearance, background, border, cursor, height, left, position, top, width

Reply:
Oof, yeah. I meant to fix this in 723455e but apparently I fail at alphabetization 😅

Reply:
> Oof, yeah. I meant to fix this in [723455e](https://github.com/18F/identity-idp/commit/723455e483cd5ad05e06ed92021444ac14e0fa8e) but apparently I fail at alphabetization 😅

Fixed (I hope 🤞 ) in 3ac5ca9.